### PR TITLE
Fixed the problem with permissions and roles

### PR DESCRIPTION
**Pull request by Zastinian**
Fixed the problem caused by updating spatie dependencies.

### CODE REVIEW

**Comment by Zastinian** [comment]
I just fixed the design and the error that the roles were not shown in the admin section.

**Comment by 1day2die** [comment]
LGTM. If ice approves it can be merged

File: app/Http/Controllers/Admin/RoleController.php [hunk 1]
```diff
@@ -72,7 +72,8 @@ public function store(Request $request): RedirectResponse
         ]);
 
         if ($request->permissions) {
-            $role->givePermissionTo($request->permissions);
+            $collectedPermissions = collect($request->permissions)->map(fn($val)=>(int)$val);
```
Comment (IceToast) [inline app/Http/Controllers/Admin/RoleController.php:75]: Why need the map items to be casted to an int?

Comment (Zastinian) [reply app/Http/Controllers/Admin/RoleController.php:75]: https://spatie.be/docs/laravel-permission/v6/upgrading#content-upgrading-from-v5-to-v6

Screenshot_20240726_064658_com android chrome

Comment (1day2die) [reply app/Http/Controllers/Admin/RoleController.php:75]: @IceToast

**Pull request merged**